PC02_Graffiti_EPPERSON.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Max Epperson
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
```

Comment:
Define Turtles, variables, and add dot notation to opening lines
```suggestion
#=======Add your code here======
jarTurt = Turtle() #Turtle for piss jar and piss
platTurt = Turtle() #Turtle for platter and hand
bubTurt = Turtle() #Turtle for speece bubble 
txtTurt = Turtle() #Turtle for text 
handTurt = Turtle() #Turtle for hand
manTurt = Turtle() #Turltle for man who needs to go peepee
dribTurt = Turtle() #pee dribs

pissyellow = (225,225,20)

#get turtle to starting position 
jarTurt.home()
jarTurt.showturtle()
jarTurt.up()
jarTurt.clear()
jarTurt.goto(120,-90)
jarTurt.clear()
```